test/files/gem001/pass-bound-orientation.ifc

```diff
@@ -0,0 +1,60 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [CoordinationView]'),'2;1');
+FILE_NAME('','2024-07-04T11:23:15',(),(),'IfcOpenShell v0.7.0-f7c03db75','IfcOpenShell v0.7.0-f7c03db75','');
```

Comment:
Probably should update your local environment to 0.8.0, no?

Reply:
haha, I'm even slow updating my own software